### PR TITLE
Actor State TTL Preview Feature

### PR DESCRIPTION
**Pull request by JoshVanL**
See: https://github.com/dapr/dapr/pull/6400

### CODE REVIEW

File: daprdocs/content/en/reference/api/actors_api.md [hunk 1]
```diff
@@ -75,10 +75,18 @@ Persists the change to the state for an actor as a multi-item transaction.
...
+#### TTL
+
+With the [`ActorStateTTL` feature enabled]]({{< ref
+"support-preview-features.md" >}}), actor clients can set the `ttlInSeconds`
+field in the transaction metadata to have the state expire after that many
+seconds. If the `ttlInSeconds` field is not set, the state will not expire.
+
+Keep in mind when building actor applications with this feature enabled;
+currently, all actor SDKs will preserve the actor state in their local cache
+even after the state has expired. This means that the actor state will not be
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/actors_api.md]: When you say "reloaded from the state store until the actor is restarted or deactivated", do you mean "actor state will not be removed from the local cache if the TTL has expired until the actor is restarted or deactivated"

**Review by msfussell — COMMENTED** [review]
Left a comment.

File: daprdocs/content/en/reference/api/actors_api.md (hunk 1 above)
Comment (JoshVanL) [reply daprdocs/content/en/reference/api/actors_api.md]: The second! I'll update

File: daprdocs/content/en/reference/api/actors_api.md [hunk 2]
```diff
@@ -75,10 +75,18 @@ Persists the change to the state for an actor as a multi-item transaction.
...
-grow indefinitely.
+#### TTL
+
+With the [`ActorStateTTL` feature enabled]]({{< ref
+"support-preview-features.md" >}}), actor clients can set the `ttlInSeconds`
+field in the transaction metadata to have the state expire after that many
+seconds. If the `ttlInSeconds` field is not set, the state will not expire.
+
+Keep in mind when building actor applications with this feature enabled;
+currently, all actor SDKs will preserve the actor state in their local cache
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/actors_api.md]:
```suggestion
currently, all actor SDKs preserve the actor state in their local cache
```

File: daprdocs/content/en/reference/api/actors_api.md [hunk 3]
```diff
@@ -75,10 +75,18 @@ Persists the change to the state for an actor as a multi-item transaction.
...
+even after the state has expired. This means that the actor state will not be
+removed from the local cache if the TTL has expired until the actor is restarted
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/actors_api.md]:
```suggestion
removed from the local cache if the TTL has expired until the actor is restarted or deactivated. This behaviour will be changed in a future release
```

File: daprdocs/content/en/reference/api/actors_api.md [hunk 4]
```diff
@@ -75,10 +75,18 @@ Persists the change to the state for an actor as a multi-item transaction.
...
+or deactivated. This behaviour will be changed in a future release.
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/actors_api.md]:
```suggestion
```

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**